index.rst

```diff
@@ -4,9 +4,173 @@ Collimated Beam Projector Installation and ComCam Testing
+
+CBP Design
+==========
+
+The CBP, a Schmidt reflector, has a focal length of 625 mm and an aperture of 240 mm. It has slots for five different masks. We use an Ekspla NT242 tunable laser fiber fed into a 6" integrating sphere to illumminate the focal plane. A NIST-calibrated photodiode in the integrating sphere is read out by an electrometer and can then be used to determine the amount of flux sent to the Simonyi Telescope (see `SITCOMTN-106 <https://sitcomtn-106.lsst.io/#time-synchronization-for-the-cbp-calibration-system>`_).
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 0
+
+      * - .. image:: images/CBP_preshipment.png
+            :width: 300px
+        - .. image:: images/mask_holders.png
+            :width: 300px
+
+   **Figure 1**: The CBP in the lab in Tuscon before shipment (left) and the CBP mask holder (right).
+ 
+CBP Repair
+==========
+
+The CBP arrived on the summit badly damaged. The dew shield that held the Schmidt corrector fell off during transit, many screws came loose, the mask changer stage drive arm broke, the azimuth drive shaft was damaged, the azimuth and elevation disks were damaged, and there were metal shavings and sawdust everywhere. 
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 27 33 40
+      :header-rows: 0
+
+      * - .. image:: images/damage_body.png
+            :width: 200px
+        - .. image:: images/damage_screws.png
+            :width: 200px
+        - .. image:: images/damage_mask_changer.png
+            :width: 300px
+
+   **Figure 2**: The CBP arrived on summit damaged.  From left to right, the dew shield fell off, screws came out, and the mask changer stage arm broke.
+
+This damage is consistent with one hard drop during shipping and with the CBP crate having spent many hours on a truck on gravel roads with poor suspension.  Fortunately, although the planar mirror was superficially scratched, there was no major damage to the optics, and thus we were able to repair the CBP. We transported the CBP back down to La Serena, where we disassembled the CBP. The machine shops in La Serena and Tucson made replacements for the broken parts, and added many new holes for screws both to replace the stripped holes and to make the CBP more secure in case of future earthquakes. We finished the repair in La Serena before transporting it to the summit.
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 100
+      :header-rows: 0
+
+      * - .. image:: images/CBP_dome_front.png
+            :width: 400px
+   
+   figure:: images/CBP_dome_front.png
+   :width: 400px
+
+   **Figure 3**: The repaired CBP. Notice that there are many more screws visible around the dew shield.
+
+
+CBP and Laser Installation on Dome
+==================================
+
+Once the CBP was repaired, it was driven to the summit and then after functional tests and throughput measurements with the laser, it was installed on the dome.
+
```

Comment:
The CBP, with a custom lifting fixture designed by the vendor, was lifted by the main dome overhead crane from L5 to L7. From there, rigging was attached to supports on the ceiling of the dome. Using the Tirak hoist, the CBP was lifted up to the upper platform beside the calibration screen structure. It was then pulled into place and secured to the platform with 4 bolts. Similarly, the laser enclosure was installed on the lower platform.